packages/react-native-web/src/exports/Image/__tests__/index-test.js

```diff
@@ -248,8 +248,8 @@ describe('components/Image', () => {
       });
     });
 
-    test('is not set immediately if the image has not already been loaded', () => {
-      const uri = 'https://google.com/favicon.ico';
+    test('is set immediately while images is loading and there is no default source', () => {
+      const uri = 'https://google.com/not-yet-loaded-image.ico';
       const source = { uri };
       const { container } = render(<Image source={source} />);
```

Comment:
Perhaps this was originally true, and source was not rendered until loaded, but if you inspect the snapshot you can see that `<img />` with the source was rendered, so I renamed the test

We do render the source while loading, unless there's a default source
https://github.com/necolas/react-native-web/blob/e8098fd029102d7801c32c1ede792bce01808c00/packages/react-native-web/src/exports/Image/index.js#L210-L211